language/src/ast_to_rustspec.rs

```diff
@@ -927,6 +927,10 @@ fn translate_expr(
             sess.span_rustspec_err(e.span, "error expressions are not allowed in Hacspec");
             Err(())
         }
+        ExprKind::ConstBlock(_) => {
```

Comment:
Nice, I didn't even know what a `ConstBlock` was, surely an addition to the latest nightly ?

Reply:
looks like it 🙂